### PR TITLE
Confluence: add basic loader

### PR DESCRIPTION
**Pull request by allisonbraithw**
Webhook configuration looks like a nightmare so we're going to sync on a cadence

still to do:
- [x] verify works
- [ ] sol'n for version-- separate index you can drop? run by meta info?

### CODE REVIEW

File: pulse/loaders/confluence.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+from pulse.vectorstore import vectorstore
+
+logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
+logger = logging.getLogger("pulse.loaders.confluence")
+
+def load_all_labelled_docs():
+    """
+    Load all documents from a confluence space with "pulse" label into the vector store
+    """
+    # rest call to confluence with the username and api key
```
Comment (allisonbraithw) [inline pulse/loaders/confluence.py:17]: could also just ingest everthing, problem is if there's too much html/markup stuff it doesn't really parse properly

Comment (allisonbraithw) [reply pulse/loaders/confluence.py:17]: maybe we don't care

File: pulse/services/confluence.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import sys
+
+from atlassian import Confluence
+from pulse import config
+
+logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
+logger = logging.getLogger("pulse.services.confluence")
+
+client = Confluence(
+    url="https://rose-garden.atlassian.net",
```
Comment (allisonbraithw) [inline pulse/services/confluence.py:12]: should also probably be config

Comment (Fraser-Isbester) [inline pulse/services/confluence.py:12]: paramaterize?

**Review by Fraser-Isbester — APPROVED** [review]

File: pulse/loaders/confluence.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    docs = confluencefactory.client.get_all_pages_from_space(config.confluence_space, expand="body.view,version")
+    for doc in docs:
+        doc_loader(doc, vectorstore)
+        
+    return docs
+
+def doc_loader(doc, vectorstore: VectorStore):
```
Comment (Fraser-Isbester) [inline pulse/loaders/confluence.py:24]: Return type!

**Review by Fraser-Isbester — APPROVED** [review]
Looks good! Response typing would be cool though!

File: pulse/loaders/confluence.py (hunk 3 above)
Comment (allisonbraithw) [reply pulse/loaders/confluence.py]: tbh don't know what it is & don't really need to return anything sooooo

**Review by Fraser-Isbester — APPROVED** [review]

**Pull request merged**